### PR TITLE
Refactor investment filter forms to use a simple form builder

### PR DESCRIPTION
**Pull request by tyom**
- Create a new builder to make filters form from a JS configuration with combination of labels file
- Add new macro which expects the macro configuration build by the builder above
- Add tests for builders

### CODE REVIEW

File: src/apps/investment-projects/builders.js [hunk 1]
```diff
@@ -66,7 +67,23 @@ function buildInvestmentSorting (originalQuery = {}) {
   }
 }
 
+function buildMacroMacroConfigFromFormFields (filtersQuery = {}) {
```
Comment (teneightfive) [inline src/apps/investment-projects/builders.js]: `MacroMacro`?

Comment (teneightfive) [inline src/apps/investment-projects/builders.js]: `FromForm`?

File: src/apps/investment-projects/builders.js [hunk 2]
```diff
@@ -66,7 +67,23 @@ function buildInvestmentSorting (originalQuery = {}) {
...
+  if (!filterFields) { return null }
```
Comment (teneightfive) [inline src/apps/investment-projects/builders.js]: Where is `filterFields ` coming from?

File: config/nunjucks/filters.js [hunk 3]
```diff
@@ -100,6 +101,23 @@ const filters = {
     }
     return result
   },
+
+  applyMacro (config) {
```
Comment (teneightfive) [inline config/nunjucks/filters.js]: Could there be a way to make use of [callAsMacro](https://github.com/uktrade/data-hub-frontend/blob/67fa79ad1c0326274afd64166c5c331b4cac2ada/config/nunjucks/globals.js#L10-L16) from globals instead?

Comment (teneightfive) [inline config/nunjucks/filters.js]: Could it be a function in global to keep these kind of actions in the same place? Might be easier for someone to see that there are 2 ways of calling macros using a variable if they're in the same place.

Comment (tyom) [inline config/nunjucks/filters.js]: Moved it to globals.

**Pull request merged**